oss_audit/tools/licensetool.py

```diff
@@ -139,8 +139,10 @@ def main():
     )
     parser.add_argument("url", help="Jar file location url")
     parser.add_argument("output", help="Output file")
+    parser.add_argument("--log_level", required=False, default="ERROR",help="Provide logging level. Example --log_level debug', default='warning'")
```

Comment:
```suggestion
    parser.add_argument("--log_level", required=False, default="ERROR",help="Provide logging level. Example --log_level debug")
```
Does this look okay? Perhaps there is an extra `default='warning'` in there.

Reply:
that's better.  licensetool.py only uses info/debug.  changed it to WARNING.  python throws if its not all caps.

Reply:
Looks good. Thanks